b

```diff
@@ -1,40 +1,23 @@
-#!/usr/bin/env sh
-set -e
+#!/usr/bin/env bash
+set -euxo pipefail
 
 TAG=$(git describe --match=NeVeRmAtCh --always --abbrev=8 --dirty)
 ORG="jitolabs"
 
-if [ "$(uname)" = "Darwin" ]; then
-    RPC_SERVERS=http://docker.for.mac.localhost:8899
-    WEBSOCKET_SERVERS=ws://docker.for.mac.localhost:8900
-    BLOCK_ENGINE_AUTH_SERVICE_URL=http://docker.for.mac.localhost:1005
-    BLOCK_ENGINE_URL=http://docker.for.mac.localhost:1000
-  elif [ "$(expr substr $(uname -s) 1 5)" = "Linux" ]; then
-    RPC_SERVERS=http://172.17.0.1:8899
-    WEBSOCKET_SERVERS=ws://172.17.0.1:8900
-    BLOCK_ENGINE_AUTH_SERVICE_URL=http://172.17.0.1:1005
-    BLOCK_ENGINE_URL=http://172.17.0.1:1000
-  else
-    echo "unsupported testing platform, exiting"
-    exit 1
-  fi
+export DOCKER_LOCALHOST=172.17.0.1
+if [[ $(uname) == "Darwin" ]]; then
```

Comment:
wanna add back the else that exits if unsupported platform?

Reply:
what other platforms would you like to guard against? windows cant run bash scripts without wsl, and wsl presents itself as linux. this is a direct copy from block engine

Reply:
I'm just referring to this: https://github.com/jito-foundation/jito-relayer/pull/69/files#diff-3e23e8160039594a33894f6564e1b1348bbd7a0088d42c4acb73eeaed59c009dL17

Reply:
added check